static/app/views/onboarding/onboarding.spec.tsx

```diff
@@ -144,10 +144,10 @@ describe('Onboarding', function () {
   });
 
   it('renders SDK data removal modal when going back', async function () {
-    const vueProject: Project = TestStubs.Project({
-      platform: 'javascript-vue',
+    const reactProject: Project = TestStubs.Project({
```

Comment:
this was originally react and I updated when migrating the react docs. This PR definitely fixes it as all javascript docs will no longer fetch any docs. Without these changes the tests won't pass